frontend/src/components/Signup/Signup.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { Row, Container, Col } from 'react-bootstrap';
+import { Animated } from 'react-animated-css';
+import { Input, Icon, Button } from 'antd';
+import './Signup.css';
+
+const circleLogo = require('../../assets/imgs/circle-logo.svg');
+
+class Signup extends Component {
+  state = {
+    email: '',
+    profile: '',
+    password1: '',
+    password2: '',
+    firstName: '',
+    lastName: '',
+    address: '',
+    phoneNumber: '',
+    dateOfBirth: ''
+  }
+
+  handleChange = (e) => {
+    this.setState({[e.target.name]: e.target.value});
+  }
+
+  render(){
+    return(
+      <div className="signupBG">
+        <Animated animationIn="fadeInLeft" isVisible={true}>
+          <Container fluid className="removePadding">
+            <Row className="removeMargin">
+              <Col xs lg="5" className="removePadding suLeft">
+                <div className="logoBox">
+                  <img src={circleLogo}></img>
+                </div>
+              </Col>
+              <Col className="removePadding">
+                <div className="formBox">
```

Comment:
Let's use the `Form` component from `antdesign`. That way we have a way to know when the user clicks submit and we don't need to always set state every time the user types something. You can refer to the `Login` component.